packages/vaex-core/vaex/functions.py

```diff
@@ -146,6 +146,16 @@ def fillna(ar, value, fill_nan=True, fill_masked=True):
 
 ########## datetime operations ##########
 
+
+def _pandas_dt_fix(x):
+    # see https://github.com/pandas-dev/pandas/issues/23276
+    import pandas as pd
+    # not sure which version this is fixed in
```

Comment:
The fix came in 0.24 ;)

Reply:
I thought so! I had a test on this, so it didn't copy it for 0.24, but the tests failed. Then I saw I had version 0.24.1 and @JovanVeljanoski (who reported the issue) had 0.24.2.
In case you want to reproduce, comment out line 154-155 and run
```
py.test tests/datetime_test.py
```
Thanks for keeping an eye on this!